### DISCUSSION TITLE
JSonish schemas

### DISCUSSION THREAD

**Discussion by unicomp21** [google/filament#4950] [Ideas]
Is there some sort of schema for the JSonish configs?  The reason I ask, wondering if it would be useful to have a set of typescript interfaces for generating json at runtime, which would then be converted to jsonish?  Having typescript code completion for this sort of thing could be helpful to new folks and learners?

**Comment by romainguy** [comment]
We don't have a schema but it would be easy to write one. Note that you don't need a json to jsonish step, pure json should work.

**Comment by unicomp21** [comment]
In that case, all we need is just typescript interfaces describing the json, right?
